files/config/default/etc/workflows/uct-publish-fast.xml

```diff
@@ -0,0 +1,430 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>fast</id>
+  <title>UCT - Publish fast</title>
+  <tags>
+    <tag>archive</tag>
```

Comment:
If this workflow is only run by Opencast Studio uploads then we don't need it to be runnable from Start task in Admin UI so I think we could remove this archive tag.